plugins/amd-kria/README.md

```diff
@@ -0,0 +1,39 @@
+# AMD Kria
+
+The AMD Kria plugin is used to represent the system firmware stored on QSPI
+for the AMD Kria system on module device specifically when not booted using
+UEFI support in U-Boot.  **When UEFI support is used the plugin will be disabled.**
+
+It uses the devices created by the mtd plugin to discover the firmware
+version and uses the known behavior of U-Boot ESP handling to distribute
+updates to the device.
+
+U-Boot will automatically pick up the firmware (so no efivars are needed) and
+will also clean up the firmware after upgrade is completed.
+
+## GUID Generation
+
+These devices use a GUID generation scheme that reflects data stored in the
+EEPROM on the board the Kria SoM is inserted into.
+
+* `KRIA\VENDOR_XILINX`
```

Comment:
KRIA as a prefix? It's usually a subsystem -- maybe UEFI in this case

Reply:
Sure uefi makes sense.